workbench-core/datasets/src/dataSetService.test.ts

```diff
@@ -623,7 +648,7 @@ describe('DataSetService', () => {
     let plugin: S3DataSetStoragePlugin;
 
     beforeEach(() => {
-      service = new DataSetService(audit, log, metaPlugin);
+      service = new DataSetService(audit, log, metaPlugin, authzPlugin);
```

Comment:
looks like new DataSetService is created every time a test runs, move this to a beforeEach on the larger DataSetService scope. Reduce the need to have a beforeEach in every sub-scope

Reply:
fixed.